docs/tasks/early_adopter_upgrade.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 2
+---
+
+# Upgrade Instructions for Early Adopters
+
+If you were part of the early adopter programme and using Rancher Turtles v0.5.x or less then you need to follow these instructions to use the GA version.
+
+:::note
+The early adopter programme was for non-production use. If you where using Turtles in a production, please contact Rancher support before upgrading.
+:::
+
+## Overview
+
+To upgrade to the GA version of Rancher Turtles you will need to do the following:
+
+1. Delete any child CAPI cluster definitions
+2. Helm uninstall Rancher Turtles
+3. Clean-up orphaned resources
+4. Install the GA version
+
+## Steps
+
+:::danger
+These steps are destructive to child clusters created using CAPI. If you have any concerns, contact Rancher support before proceeding.
+:::
+
+1. Delete any CAPI child cluster definitions and wait for CAPI to fully delete the child clusters
+2. Delete the CAPI providers installed:
+
+```bash=
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-kubeadm-control-plane-system kubeadm-control-plane
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-kubeadm-bootstrap-system kubeadm-bootstrap
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-system cluster-api
+kubectl delete ns capi-kubeadm-control-plane-system
+kubectl delete ns capi-kubeadm-bootstrap-system
+kubectl delete ns capi-system
```

Comment:
2 comments on these:
1. we might need to extend it to add rke2 providers
2. Deleting namespaces might result in them going into "Terminating" state since they have finalizers and stay forever

Reply:
Not for the early adopters as the RKE2 provider will only default from the GA version.

Reply:
I tested the namespaces and they delete as we are deleting the clusters and CAPIProviders first.

Reply:
>Not for the early adopters as the RKE2 provider will only default from the GA version.

Right, makes sense. Once we enable it by default in https://github.com/rancher/turtles/pull/437, we can update it